aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/utils/VersionInformation.java

```diff
@@ -0,0 +1,511 @@
+package com.castsoftware.aip.console.tools.core.utils;
+
+import com.castsoftware.aip.console.tools.core.exceptions.InvalidVersionException;
+import jdk.nashorn.internal.runtime.regexp.joni.exception.InternalException;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+
+/**
+ * Object representing a version composed of 2 to 4 numbers separated by a dot.
+ */
+public final class VersionInformation {
```

Comment:
i prefer do not manage this matrix, just ignore the error if debug option doesnt exist in console < 1.24
if we manage the matrix, we need to also manage it CLI syntax and interface of jenkins

Reply:
But how I compare version then if I can't detect in which version it was added.
We already use VersionInformation to compare between versions, so was a good candidate.
If not I'll just catch the exception and forget about it

Reply:
yes, remove everything,  just catch the exception, i didnt expect you to manage the version matrix in this fix.
the job has been failed for one week

Reply:
Here is the major problem ignoring the exception: if for any reason the feature exists and failed to fetch debug option then we will put it "under the carpet" (like they say in french.

Reply:
i said already, if u want to manage this, you need to manage the UI of jenkins and CLI syntax,  u can keep this branch if u want and create another branch to just fix the pb

Reply:
ok I'd like to do so.
But this branch is just to fix the test and can be used out of the box